.github/workflows/docker-build.yml

```diff
@@ -4,29 +4,54 @@ on:
   push:
     branches:
       - master
+  release:
+    types:
+      - published
 
 jobs:
   publish:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Get Release Tags
+        id: get_release_tags
+        run: |
+          echo "::set-env name=RELEASE_TAG::$(echo ${GITHUB_REF:10})"
+          echo "::set-env name=TAG::stable"
+          echo "::set-env name=TAG_SEMVER::true"
+        if: github.event_name == 'release'
+
+      - name: Get Push Tags
+        id: get_push_tags
+        run: |
+          echo "::set-env name=TAG::latest"
+          echo "::set-env name=RELEASE_TAG::$(echo ${GITHUB_REF:11})"
```

Comment:
Just to confirm, this will get the number only (remove the leading `v` from the release tag), right?

As an example the release tag is `v2.1.1` whereas the docker image tag is `2.1.1` for the 2.1.1 release.

Reply:
No, it will not do that. But the action has an option to set `TAG_SEMVER`, which will automatically tag all semantic versions accordingly if 1.0.0 then 1, 1.0, 1.0.0 are the three tags set. The Push tag should be `refs/branches/master` and `latest`.

Reply:
Upon closer inspection of the action code, You can either use custom tags or `semver tags`, but not both. 
https://github.com/elgohr/Publish-Docker-Github-Action/blob/master/entrypoint.sh#L70-L86

Reply:
> No, it will not do that. But the action...

Wait, so this will not follow the tagging convention when building the release docker images? Could this be updated to follow the conventions?